src/host/util/PersistentStorage.h

```diff
@@ -65,15 +65,37 @@ class PersistentStorage {
       };
     };
 
+    struct BNO055CalOffsets {
+      uint16_t accel_offset_x;
+      uint16_t accel_offset_y;
+      uint16_t accel_offset_z;
+      uint16_t gyro_offset_x;
+      uint16_t gyro_offset_y;
+      uint16_t gyro_offset_z;
+      uint16_t mag_offset_x;
+      uint16_t mag_offset_y;
+      uint16_t mag_offset_z;
+      uint16_t accel_radius;
+      uint16_t mag_radius;
+    };
+
+    struct IMUHeelPitchOffsets {
+      int offsetHeel;
+      int offsetPitch;
+    };
+
   private:
     // Addresses of elements in flash storage
     static const uint16_t magicAddress = 0;
     static const uint16_t versionAddress = 4;
     static const uint16_t nmea2000ParamsAddress = 8;
+    static const uint16_t bno055CalOffsetsAddress = 20;     // 11 x 2 Byte
```

Comment:
I will replace this by `nmea2000ParamsAddress + sizeof(NMEA2000Params)` this way it will always be correct even if we change the size of the `NMEA2000Params`. Let me know if you see any problem with this.

Of course we will need to upgrade the version too when we do that so that old data is considered invalid.

Reply:
Aah, now I know why you did the version!

The disadvantage could be:
Other values on higher addresses stored, e.g. Logging of total NM will get lost too
And you will have more writing-processes at the end.

May be there should be space reserved for permanent values or clear values which have fixed size and you put things which are not clear yet to higher addresses?

Reply:
yeah right now it does not handle that well.